projects/core/ng-package.json

```diff
@@ -8,7 +8,8 @@
       "@ngrx/router-store": "fromNgrxRouter",
       "@ngrx/store": "store",
       "i18next": "i18next",
-      "i18next-xhr-backend": "i18nextXhrBackend"
+      "i18next-xhr-backend": "i18nextXhrBackend",
+      "@spartacus/incubator": "incubator"
```

Comment:
this looks like a really bad idea. We should not make a dependency from core to incubator, as all production  customers are forced to pull the incubator. Or am I missing something?